test/UnbrandedProjects/Platform-OpenAI-TypeSpec/src/Generated/OpenAIClientOptions.cs

```diff
@@ -0,0 +1,13 @@
+// <auto-generated/>
+
+#nullable disable
+
+using System.ServiceModel.Rest;
+
+namespace OpenAI
+{
+    /// <summary> Client options for OpenAIClient. </summary>
+    public partial class OpenAIClientOptions : RequestOptions
```

Comment:
I didn't see any version information in this class, so is it by design for the unversioned service?

Reply:
It doesn't define a version in the spec so there is no version in the generated code correct.